index.html

```diff
@@ -8,6 +8,54 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+  <header class="header">
+    <div class="top-header">
+      <div class="time"><p>9.41</p></div>
+      <div class="wifi-net-battery">
+        <img src="./images/Cellular.svg" class="icon" alt="twitter icon">
+        <img src="./images/Wifi.svg" class="icon" alt="twitter icon">
+        <img src="./images/Battery.svg" class="icon" alt="twitter icon">
+      </div>
+    </div>
+
+    <div class="bottom-header">
+      <div class="logo-section">
+        <h1 class="logo">Rose</h1>
+      </div>
+      <div class="bars">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+    </div>
+    
+  </header>
+  <main class="main-content">
+    <div class="hero-section">
+      <div class="about-title-section">
+        <h2 class="about-title padding">I'm Rose Glad to see you!</h2>
+      </div>
+      <div class="full-description">
+        <p class="about-me padding">I’m a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like what you see
+          and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <div class="socials-section">
+        <button class="connect-link padding">LET'S CONNECT</button>
+        <div class="icons">
+          <img src="./images/twitter.svg" class="icon" alt="twitter icon">
+          <img src="./images/Linkedin.svg" class="icon" alt="twitter icon">
+          <img src="./images/mail.svg" class="icon" alt="twitter icon">
+          <img src="./images/github.svg" class="icon" alt="twitter icon">
+          <img src="./images/playboy.svg" class="icon" alt="twitter icon">
+        </div>
```

Comment:
- [ ] Using `li` and `ul` is not only more semantic, but it also gives meaning to the listed element and helps the screen readers identify the important part, Please consider using `ul` element instead of `div`.